common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3204,6 +3213,22 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(32)
 
+  val MEMORY_POOLED_ALLOCATOR_VERBOSE_METRIC: ConfigEntry[Boolean] =
+    buildConf("celeborn.pooled.allocator.verbose.metric")
```

Comment:
checked `PooledByteBufAllocator`, netty uses `io.netty.allocator.*`, can we use `celeborn .memory.allocator.*` and align the properties w/ netty? e.g. 
`io.netty.allocator.numHeapArenas` => `celeborn .memory.allocator.numHeapArenas`

Reply:
Sounds reasonable. I'll change it to make it looks consistent with Netty.